packages/Python3/CONFIG

```diff
@@ -0,0 +1,144 @@
+Python3_VERSION ?= 3.8.0
+
+MD5_SUM_3.8.0 = 3689a67099dfaf3c9077293ef1f3e21e
+
+TAR_EXTENSION = .tar.gz
+
+PYTHON3_PYC_ONLY ?= 0
+
+
+# All the numbered patches below were taken from buildroot.
+PATCHES_3.8.0 = \
+    0001-Make-the-build-of-pyc-files-conditional.patch \
+    0002-Disable-buggy_getaddrinfo-configure-test-when-cross-.patch \
+    0003-Add-infrastructure-to-disable-the-build-of-certain-e.patch \
+    0004-Adjust-library-header-paths-for-cross-compilation.patch \
+    0005-Don-t-look-in-usr-lib-termcap-for-libraries.patch \
+    0006-Don-t-add-multiarch-paths.patch \
+    0007-Abort-on-failed-module-build.patch \
+    0008-Serial-ioctl-workaround.patch \
+    0009-Do-not-adjust-the-shebang-of-Python-scripts-for-cros.patch \
+    0010-Misc-python-config.sh.in-ensure-sed-invocations-only.patch \
+    0011-Override-system-locale-and-set-to-default-when-addin.patch \
+    0012-Add-importlib-fix-for-PEP-3147-issue.patch \
+    0013-Add-an-option-to-disable-installation-of-test-module.patch \
+    0014-Add-an-option-to-disable-pydoc.patch \
+    0015-Add-an-option-to-disable-lib2to3.patch \
+    0016-Add-option-to-disable-the-sqlite3-module.patch \
+    0017-Add-an-option-to-disable-the-tk-module.patch \
+    0018-Add-an-option-to-disable-the-curses-module.patch \
+    0019-Add-an-option-to-disable-expat.patch \
+    0020-Add-an-option-to-disable-CJK-codecs.patch \
+    0021-Add-an-option-to-disable-NIS.patch \
+    0022-Add-an-option-to-disable-unicodedata.patch \
+    0023-Add-an-option-to-disable-IDLE.patch \
+    0024-Add-an-option-to-disable-decimal.patch \
+    0025-Add-an-option-to-disable-the-ossaudiodev-module.patch \
+    0026-Add-an-option-to-disable-openssl-support.patch \
+    0027-Add-an-option-to-disable-the-readline-module.patch \
+    0028-Add-options-to-disable-zlib-bzip2-and-xz-modules.patch \
+    0029-python-config.sh-don-t-reassign-prefix.patch \
+    0030-Fix-cross-compiling-the-uuid-module.patch \
+    0031-Add-an-option-to-disable-uuid-module.patch \
+    0032-fix-building-on-older-distributions.patch \
+    0033-configure.ac-fixup-CC-print-multiarch-output-for-mus.patch \
+    1000-no-implicit-function-declaration-as-error.patch
```

Comment:
I am so queasy about this patch, it hides such a multitude of sins.  However, Emilio tells me that we already need the equivalent for Python2!

As I understand it, this is what is happening.  The target `./configure` script checks for (amongst other things) the presence of `getentropy`, and successfully finds it ... despite this function *not* being present in our libc or declared in *any* header file.  It turns out that `libressl` creates its own fallback implementation, which is picked up at link time, but not declared.  In effect this is a private implementation for `libressl` which Python3 almost accidentally just picks up.  Damn.

Actually, it turns out that `libressl` defines lots and lots of fallback implementations for its own private use, and uses the gcc `#include_next` extension to insert these definitions in front of the standard libc headers.  However, it doesn't look like these extension headers are installed in the staging area, so we probably don't want to try using them.

In brief, I guess we need to keep this (or maybe just override the config option if that works).